### PR TITLE
Pranitha's clicky turtle reflection.

### PR DESCRIPTION
**Pull request by pranithavallabhaneni**

### CODE REVIEW

File: posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout :  post
+author :  pranithavallabhaneni
+title  : "Pranitha's clicky turtle reflecttion "
+---
+For this clicky turtle hack my thoughts was to create a different trapeziums by click and to change back ground colors by using arrow keys. 
+
+**Difficulties:**
+First difficulty I faced was during asssigning colors to the arrow keys .I initially don't know where to start and later I used *myscreen.onkey function and tried multiple times to get it.
```
Comment (haley-triem) [inline posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md:9]: Quick typo with the period after keys--looks like there might be an extra space!

File: posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+myscreen.onkey(blue_trapezium, 'right')
+myscreen.onkey(violet_trapezium, 'up')
+myscreen.onkey(red_trapezium, 'down')
+```
+Second difficulty was during creating multiple trapeziums by clicking and here the mistake I made was that I gave the clicky functin to (x,y) co-ordinates but I forgot to give to trapezium the clicky function and later on I corrected my mistake and did it in the correct way.you can see the code below.
+```
+def clicky(x, y):
+ trapezium(x,y)
+ ```
+ Finally after many attempts I managed to get the code that I required without any errors.I listed my clicky turtle program down below in the link you can have a look in to it.
```
Comment (haley-triem) [inline posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md:21]: Quick typo after errors: looks like a space got dropped between the period and "I"

**Review by haley-triem — COMMENTED** [review]
Hi Pranitha, your code is looking good; here are some quick notes before I merge:

- File: I think you might have mistyped the file placement when first making your post ... it should be "_posts" instead of "posts". ... I also think you might have accidentally put the post name for 10/5 instead of 10/6.

- I think under `title:` there's a typo: it looks like "reflecttion" has two t's on accident!

- On line 9, I think it would be more readable to have "myscreen.onkey()" in backticks, like you do for the bigger chunks of code (so it would look more like `myscreen.onkey()`

File: posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md (hunk 1 above)
Comment (pranithavallabhaneni) [reply posts/pranithavallabhaneni/10-05-2022-pranithavallabhaneni-clicky-turtle-reflection.md]: Thanks for the suggestions haley. I have corrected the typos and submitted it . can you review it again when you got free time.Thank you:)

**Review by haley-triem — APPROVED** [review]
I think the changes help a lot! I will go ahead and approve now. Thanks!

**Pull request merged**